feature/repair/src/main/java/com/iteneum/repair/presentation/RepairView.kt

```diff
@@ -0,0 +1,158 @@
+package com.iteneum.repair.presentation
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringArrayResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.iteneum.designsystem.components.*
+import com.iteneum.designsystem.theme.LPTypography
+import com.iteneum.designsystem.theme.LeasePertTheme
+import com.iteneum.repair.R
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Preview
+@Composable
+fun RepairView() {
+    val dp24 = LeasePertTheme.sizes.regularSize
```

Comment:
I recommend you use just one variable for the sizes container `val sizes = LeasePertTheme.sizes` and then use the sizes in your code to get all the ones that you need. For example if you need different extraSizes and also the midSmall size you use do the following:
`sizes. extraSize124`
`sizes. extraSize128`
`sizes. midSmallSize`
`sizes. minorRegularSize`
it is just to simplify the code and not create an each variable for each size